docs/guides/migrating-to-7.0.md

```diff
@@ -4,7 +4,7 @@ title: Migrating to v7.0 - Sinon.JS
 breadcrumb: migrating to 7.0
 ---
 
-For upgrading to new Sinon 7 there are no known major breaking Changes except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
+For upgrading to new Sinon 7 there is no known major breaking Change except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
```

Comment:
I didn't object to the plural, but the letter casing :smile:

Reply:
My Bad I could not understand your fat Arrow Function. 😆.